numpy/lib/nanfunctions.py

```diff
@@ -493,7 +496,11 @@ def nansum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
 
     Returns
     -------
-    y : ndarray or numpy scalar
+    nansum_along_axis : ndarray.
+        A new array holding the result is returned unless `out` is
+        specified, in which it is returned. The
+        result has the same size as `a`, and the same shape as `a` if
```

Comment:
Agreed I wondered about that myself but didn't want to depart from convention.  Fixed in `29cca2d`